.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     "airbnb-typescript",
     "plugin:@next/next/recommended",
     "prettier",
+    "plugin:react/jsx-runtime",
```

Comment:
When disabled, Eslint throws errors for missing React import, as per their [`ReadMe`](https://github.com/yannickcr/eslint-plugin-react)


> If you are using the new JSX transform from React 17, extend react/jsx-runtime in your eslint config (add "plugin:react/jsx-runtime" to "extends") to disable the relevant rules.


However as per [React's](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports) docs on this 

> React will no longer need to be in scope when you use JSX. This might lead to unused React imports in your code. It doesn’t hurt to keep them